### PR TITLE
[Uber Eats] Configuration + User Interface

### PR DESCRIPTION
**Pull request by janainawittner**

### CODE REVIEW

File: content/apps/uber-eats/en/configuration.md [hunk 1]
```diff
@@ -28,16 +28,16 @@ On the other hand, you can also customise which status to use on HubRise for new
 
 ## Special Ref Codes and Other Options
 
-This section allows you to customise several miscellaneous options:
+This section allows you to customise several miscellaneous options: // JW
```
Comment (janainawittner) [inline content/apps/uber-eats/en/configuration.md:31]: Same as Deliveroo, we need to refer to the EPOS.

File: content/apps/uber-eats/en/configuration.md [hunk 2]
```diff
@@ -28,16 +28,16 @@ On the other hand, you can also customise which status to use on HubRise for new
...
 
 - The service type ref code associated to Uber Eats orders.
 - The payment type and ref code associated with Uber Eats orders.
-- The ref code used when a customer includes disposable items in an order.
+- The ref code used when a customer includes disposable items in an order. //JW
```
Comment (janainawittner) [inline content/apps/uber-eats/en/configuration.md:35]: I think its important to explain that a **Disposable items** product should be included in the EPOS menu and this ref code is to be reported here. 

 **Disposable items** is an option clients can tick in the Uber Eats basket on checkout. Not all restaurants include this option in the basket but if they do, they need to include the ref code.

File: content/apps/uber-eats/en/configuration.md [hunk 3]
```diff
@@ -28,15 +28,15 @@ On the other hand, you can also customise which status to use on HubRise for new
...
 
 ## Saving the Configuration
 
-Once you are happy with the configuration of the Uber Eats Bridge, click **Save** at the top of the page to continue to the main page.
+Once you are happy with the configuration of the Uber Eats Bridge, click **Save** at the top of the page to continue to the main page. // JW
```
Comment (janainawittner) [inline content/apps/uber-eats/en/configuration.md:39]: Do we have another name for main page?
Perhaps bridge in

File: content/apps/uber-eats/en/configuration.md [hunk 4]
```diff
@@ -28,16 +28,16 @@ On the other hand, you can also customise which status to use on HubRise for new
...
 
 ## Resetting the Configuration
 
-You can always restore the Configuration page to its default values and change the Uber Eats store ID associated with your HubRise location by clicking on **Reset the configuration** at the bottom of the page.
+You can always restore the Configuration page to its default values and change the Uber Eats store ID associated with your HubRise location by clicking on **Reset the configuration** at the bottom of the page. // JW
```
Comment (janainawittner) [inline content/apps/uber-eats/en/configuration.md:43]: It's not finished....
They need to Map ref codes too and I cannot see any reference to it anywhere. 
Here is a cut/paste of emails I have been sending clients: https://docs.google.com/document/d/1O6v1loWCp2fxsh3np1wf5Nu1VlynUxKMIjnM2J5yjpQ/edit

There is a section with where we show how to map ref codes. We know how to do this for Uber Eats, so might as well document it.

File: content/apps/uber-eats/en/configuration.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 title: Configuration
 position: 3
 layout: documentation
-meta:
+meta: // JW
```
Comment (janainawittner) [inline content/apps/uber-eats/en/configuration.md:5]: Uber Eats Connection to HubRise - Configuration

See instructions to configure the Uber Eats Bridge to work seamlessly with Uber Eats and your EPOS or other apps connected to HubRise. Configuration is simple.

**Comment by antmonnier** [comment]
@atorin Can you address Janaina's comments + organize the "User Interface" & "Configuration" pages in a similar way to Deliveroo?

**Comment by atorin** [comment]
@antmonnier Sure, I can do that

File: content/apps/uber-eats/en/configuration.md (hunk 3 above)
Comment (atorin) [reply content/apps/uber-eats/en/configuration.md:39]: We call it "Operations page" in Deliveroo (https://www.hubrise.com/apps/deliveroo/user-interface/#operations-page)

File: content/apps/uber-eats/en/configuration.md (hunk 4 above)
Comment (atorin) [reply content/apps/uber-eats/en/configuration.md:43]: @janainawittner That's a good idea! Shall we create a separate page in the document where we explain this? 
This Configuration page does not seem the right place to put this IMO.

File: content/apps/uber-eats/en/configuration.md (hunk 1 above)
Comment (atorin) [reply content/apps/uber-eats/en/configuration.md]: @janainawittner I've added a reference to the EPOS

File: content/apps/uber-eats/en/configuration.md (hunk 2 above)
Comment (atorin) [reply content/apps/uber-eats/en/configuration.md]: @janainawittner I've added this sentence.

File: content/apps/uber-eats/en/configuration.md (hunk 3 above)
Comment (atorin) [reply content/apps/uber-eats/en/configuration.md]: @janainawittner I've used "Operations page" and used a similar wording also in Deliveroo's docs. Let me know if you prefer a different name.

File: content/apps/uber-eats/en/configuration.md (hunk 4 above)
Comment (atorin) [reply content/apps/uber-eats/en/configuration.md]: @janainawittner I didn't realise that @antmonnier had already opened issue #163 for this. 
I'll resolve this comment for now and rely on the issue for further discussions. Thanks!

**Comment by atorin** [comment]
I've modified the name of this PR to include the changes to the User Interface page as well.

File: content/apps/uber-eats/en/configuration.md (hunk 3 above)
Comment (janainawittner) [reply content/apps/uber-eats/en/configuration.md]: **Operations page** is what Antoine calls it. I have no better name :-)

Comment (janainawittner) [reply content/apps/uber-eats/en/configuration.md]: Unless you guys have a better name for it I guess you can **Resolve this conversation** :-)

**Comment by janainawittner** [comment]
I think i am all done :-)
Thank you @atorin

**Comment by atorin** [comment]
@antmonnier Please don't merge yet! I still have to modify the user interface page.

@janainawittner I'll let you know when I'm done, but it will be mostly a copy/paste from Deliveroo.

**Comment by atorin** [comment]
Now I'm done!

**Pull request merged**